src/Keboola/DbExtractor/Extractor/Extractor.php

```diff
@@ -104,6 +104,8 @@ public function __construct(array $parameters, array $state = [], ?Logger $logge
 
     abstract public function simpleQuery(array $table, array $columns = array()): string;
 
+    abstract public function getMaxOfIncrementalFetchingColumn(array $table): ?string;
```

Comment:
so this will be a major version update?

Reply:
yeah....we must release new major version